src/deploy/osp_deployer/checkpoints.py

```diff
@@ -302,14 +309,16 @@ def verify_undercloud_installed(self):
             raise AssertionError(
                 "Director & Undercloud did not install properly, "
                 "check /pilot/install-director.log for details")
-                
-        cmd = "grep \"The Undercloud has been successfully installed\" " + "~/pilot/install-director.log"
+
+        cmd = ("grep \"The Undercloud has been successfully installed\" "
+               + "~/pilot/install-director.log")
+
         setts = self.settings
         re = Ssh.execute_command_tty(self.director_ip,
                                      setts.director_install_account_user,
                                      setts.director_install_account_pwd,
                                      cmd)
-        if "The Undercloud has been successfully installed" not in re[0]:
+        if "Undercloud install complete." not in re[0]:
```

Comment:
^ Should still be "The Undercloud has been successfully installed"

Reply:
Yes, this is an error, I must have picked wrong conflict resolution during git rebase.